index.html

```diff
@@ -376,6 +376,11 @@ <h3 id="mems">MEMS and Microsensors</h3>
 <ul>
    <li>Course Notes written in Latex by <i>Francesco Lenzi & Donato Carlo Giorgio</i> - <a href="4/MEMS/MEMS_and_Microsensors.pdf" download>download</a> </li>
 </ul>
+
+<h3 id="DICD">Digital Integrated Circuit Design</h3>
+<ul>
+   <li>Course Notes by <i>Luca Colombo</i> - <a href="4/DICD/Digital Integrated Circuit Design.pdf" download>download</a> </li>
+</ul>
```

Comment:
Dovresti aggiungere DICD anche nella parte di html in alto dove vengono elencate le materie.

Così tipo:

![Screenshot (128)](https://github.com/valerionew/triennale-elettronica-polimi/assets/68781644/66c8e4e1-df35-42fa-b198-211a46b26a11)

Occhio solo all'id, nel tuo caso sarebbe **id=#DICD**